src/js/components/ErrorsAlert.js

```diff
@@ -47,7 +47,7 @@ const ErrorsAlert = function(props) {
   const errorItems = errorMessages.map((message, index) => {
     return (
       <li key={index} className="errorsAlert-listItem">
-        {message}.
+        {message}
```

Comment:
Is there a good reason for removing this? I added it so that all errors will have dots at the end, not just some of them.

Reply:
That should be enforced where the messages are generated, not where they are displayed. Adding the dot here broke the jobs form messages (they all had 2 dots)

Reply:
The thing is that some of them are generated by us and some by the server and we can't enforce the latter to have a dot.

Reply:
I'm not certain that this shared presentational component is the right place to enforce component-specific needs. IMO the text should be formatted as it needs to be before it is given to ErrorsAlert

Reply:
Not all messages are formatted by us.